cmd/influx/global.go

```diff
@@ -212,6 +229,11 @@ func coreFlags() []cli.Flag {
 			Hidden: true,
 			EnvVar: "INFLUX_TRACE_DEBUG_ID",
 		}},
+		&CommonStringSliceFlag{cli.StringSliceFlag{
```

Comment:
Would be nice to have usage, don't know if it needs to be hidden - I like the feature.